datadog_operator/assets/dashboards/operator_overview.json

```
@@ -0,0 +1,1987 @@
+                                "sum"
+                            ],
+                            "type": "timeseries",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "queries": [
+                                        {
+                                            "query": "sum:datadog.operator.workqueue_retries_total{$scope,$cluster,kube_app_name:datadog-operator} by {kube_cluster_name}.as_count()",
+                                            "data_source": "metrics",
+                                            "name": "query1"
+                                        }
+                                    ],
+                                    "style": {
+                                        "palette": "dog_classic",
+                                        "line_type": "solid",
+                                        "line_width": "normal"
+                                    },
+                                    "display_type": "bars"
+                                }
+                            ]
+                        },
+                        "layout": {
+                            "x": 4,
+                            "y": 6,
+                            "width": 4,
+                            "height": 3
+                        }
+                    },
+                    {
+                        "id": 2246786119938033,
+                        "definition": {
+                            "title": "Work Queue Adds Total",
+                            "show_legend": true,
+                            "legend_layout": "auto",
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "type": "timeseries",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "response_format": "timeseries",
+                                    "queries": [
+                                        {
+                                            "query": "avg:datadog.operator.workqueue_adds_total{$scope,$cluster,kube_app_name:datadog-operator} by {kube_cluster_name}.as_count()",
```

```suggestion
                                            "query": "sum:datadog.operator.workqueue_adds_total{$scope,$cluster,kube_app_name:datadog-operator} by {kube_cluster_name}.as_count()",
```
My reasoning being that that if we're looking at the total, then we should look at the sum of the tag combinations instead of the average of the tag combinations